ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.tsx

```diff
@@ -366,6 +367,17 @@ export class OntotextYasguiWebComponent {
     return this.getOntotextYasgui().then(() => YasqeService.showYasqeActionButtons(yasqeActionButtonNames));
   }
 
+  /**
+   * Returns info about ongoing queries and updates.
+   */
+  @Method()
+  getRanRequestsCount(): Promise<RanRequestCountInfo> {
```

Comment:
We need to be more consistent with the naming. Here the method is better to be named `getOngoingQueriesInfo` maybe. Also are we sure that we will always return only the count?